src/frequenz/dispatch/_bg_service.py

```diff
@@ -104,6 +125,11 @@ def __init__(
         always at index 0.
         """
 
+    @property
+    def dispatches(self) -> ValuesView[Dispatch]:
+        """All currently cached dispatches."""
+        return self._dispatches.values()
```

Comment:
This seems to be unused, if that is the case I would remove it to keep the public API surface as small as possible. If it is used, why not `def dispatches(self) -> Mapping[int, Dispatch]: return self._dispatches` instead of just the values?